resources/views/admin/layouts/app.blade.php

```diff
@@ -0,0 +1,42 @@
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Booking Clinic - @yield('title')</title>
+    <!-- Bootstrap core CSS -->
+    <link href="{{ asset('css/app.css') }}" rel="stylesheet">
```

Comment:
if `app.css` contains only `bootstrap` then it's better to download original bootstrap4 to `public/css/bootstrap.css`